### PR TITLE
Game Implementation Fixed

### PR DESCRIPTION
**Pull request by kyokidG**

### CODE REVIEW

File: app/static/css/game.css [hunk 1]
```diff
@@ -44,3 +34,275 @@
...
+    -webkit-box-shadow: 1px -51px 89px -3px rgba(0,255,51,1);
+    -moz-box-shadow: 1px -51px 89px -3px rgba(0,255,51,1);
+    box-shadow: 1px -51px 89px -3px rgba(0,255,51,1);
+}
+/*end of current player's tank (body+cannon) */
+
+
+/* enemy player's stats */
+#enemy-username {
+    margin: 10px;
```
Comment (luciandragomir) [inline app/static/css/game.css]: #enemy-* ids has so many duplicates with #my-* rules.. Please reduce the duplication.

File: app/static/js/game.js [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+    this.displayGravity = function (gravity) {
+        var maxGravity = 11;
+        for (i = 1; i <= maxGravity * 10; i++) {
+            var bar = document.createElement('div');
+            $(bar).addClass("bar");
+            if (i <= Math.floor(gravity) * 10) {
+                $(bar).addClass("on");
+            }
+            $(bar).height(10);
```
Comment (luciandragomir) [inline app/static/js/game.js]: Why do you need to set the height and width here?

File: app/static/js/game.js [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+            $(bar).width(5);
+            $(bar).appendTo($("#gravity-bars"));
+        }
+    }
+
+    this.startTimer = function (timer) {
+        var secs = 0;
+        setInterval(function () {
```
Comment (luciandragomir) [inline app/static/js/game.js]: When are you going to stop this timer?

File: app/static/js/game.js [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+    this.hasDefense = false;
+    this.shieldDuration = 3; //in seconds
+    this.shieldCooldwown = 2; //in seconds
+    this.shotCooldown = 1; //in seconds
+    this.totalShots = 0;
+    this.hitShots = 0;
+    this.damage = 30;
+
+    this.isDead = function () {
+        return this.currentHealth.width() <= 0
```
Comment (luciandragomir) [inline app/static/js/game.js]: Don't rely on the UI to tell you if a player is dead. The application logic should know when a player is dead, and the UI is only a way to visualize that and gather user input.

Comment (kyokidG) [inline app/static/js/game.js]: We will have the HP of each player on the server and we will check there is a player is dead, this UI validation is a temporary condition.

**Pull request merged**